index.html

```diff
@@ -0,0 +1,266 @@
+            </ul>
+    </div>
+</div>
+    </section>
+
+    <section id="wrap">
+        <div class="whitebox">
+            <div class="imageclass">
+                <img src="images/Nature.png" width="500" height="400" />
+            </div>
+            <div class="description">
+                <h2 class="heading">Tonic</h2>
+
+                <div>
+                    <ul class="list_align">
+                        <li class="bold">CANOPY</li>
+                        <img src="images/Counter.svg" />
+                        <li>Back End Dev</li>
+                        <img src="images/Counter.svg" />
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads;no accounts<br /> or sign-ups required.
+                </p>
+                <ul class="tags">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button" class="bluebutton">See More</button>
+            </div>
+        </div>
+        <div class="whitebox">
+            <div class="imageclass">
+                <img src="images/colorfulbasketball.svg" width="500" height="400" />
+            </div>
+            <div class="description">
+                <h2 class="heading">Multi-Post<br />Stories</h2>
+                <div>
+                    <ul class="list_align">
+                        <li class="bold">CANOPY</li>
+                        <img src="images/Counter.svg" />
+                        <li>Back End Dev</li>
+                        <img src="images/Counter.svg" />
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p>
+                    Experimental content creation feature that allows users to<br /> add to an existing story over the course of a day without<br /> spamming their friends.
+                </p>
+                <ul class="tags">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button" class="bluebutton">See More</button>
+            </div>
+        </div>
+        <div class="whitebox">
+            <div class="imageclass">
+                <img src="images/orange ting.svg" width="500" height="400" />
+            </div>
+            <div class="description">
+                <div>
+                    <h2 class="heading">Facebook</h2>
+                </div>
+                <div>
+                    <ul class="list_align">
+                        <li class="bold">CANOPY</li>
+                        <img src="images/Counter.svg" />
+                        <li>Back End Dev</li>
+                        <img src="images/Counter.svg" />
+                        <li>2015</li>
+                    </ul>
+                </div>
```

Comment:
- You did a great job in your design it looks nice but there is a small issue that you still need to fix to make your design 
stick to the figma design as required in the project :+1:  check the picture below to see the comparison. to fix that I suggest you use the appropriate color, font-size and add some space between those items :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/228811913-f4c1133d-2298-4b30-97df-540e56eea7fb.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/228810538-0166e0cc-990c-47ea-a0b7-b42456955696.png)